base-template/about-base-template.md

```diff
@@ -11,84 +24,349 @@ The base template goes one step further, in that it helps you create any templat
+* _[Structured data](https://developers.google.com/search/docs/guides/intro-structured-data):_
+    * _Term is used by Google._
+    * _But isn’t as evocative as “microcontent”_
+    * _Doesn’t have a wikipedia entry_
+
+{Template author tip: All templates for The Good Docs Project must describe recommended [structured data](https://developers.google.com/search/docs/guides/sd-policies) fields. Fields selected should align with equivalent doctypes used in [schema.org](https://schema.org/).}
+
+Microcontent fields are machine readable snippets of [structured data](https://developers.google.com/search/docs/guides/sd-policies) which describe their associated page. It is typically presented as concise text or an image.
+
+All technical documentation should include microcontent. It helps search engines index pages appropriately, and facilitates content reuse.
+
+Within The Good Docs Project, we have adopted the [JSON-LD](http://json-ld.org/) format for microcontent, [as recommended by Google](https://developers.google.com/search/docs/guides/intro-structured-data).
+
+Note:
+
+* Many microcontent fields need to be maintained manually which adds a maintenance burden to a project.
+* If any of the microcontent on a site is inaccurate or dated, it will reduce the trustworthiness of all the microcontent on the site, as the user typically won’t be able to determine which of the content is trustworthy.
+* As such, a project should only adopt microcontent fields which are automatically maintained by your documentation publishing system, or where there is an established, low-effort process to ensure the microcontent is maintained.
+
+## Primary goal
+
+{Template author tip: Concisely state the primary purpose of this doctype, ideally in one short sentence.}
+
+## Target audience and user stories
+
+{Template author tip: You should document the key user stories that this template is likely to be used for.}
+
+{Docset owner tip: Select the user stories which are applicable to your project.}
+
+This {doctype} template aims to address the following user stories:
+
+* As a {user persona}, I want to {achieve a specific goal}, so that I can {achieve a larger business objective}.
+
+{Template author tip: You may optionally add a “docset owner tip” explaining when the following user story should or should not be included.}
+
+* As a {user persona}, I want to {achieve a specific goal}, so that I can {achieve a larger business objective}.
+    * {Docset owner tip: Include this user story when … .}
+
+## Nature of content
+
+{Template author tip: 
+
+* Discuss the nature of the content for this doctype.
+* Consider how it will impact maintenance.
+* Is this content expressed best with multi-media, such as diagrams, screenshots or video?
+* If so, reference your style guide which should set recommendations on image and screenshot size, video length, style etc.
+
+}
+
+## Priorities for each quality criteria
```

Comment:
This section needs further refinement in my opinion. We can, for instance, better present the quality criteria and also guide the template author based on their priorities.

Reply:
Yes @viraji , I agree. I have a "TBD (To Be Decided)" comment in this section to remind us to come back to this.
I'll close this comment for the moment, and we can come back to it later.